### ISSUE TITLE
Syntax error when using ES6 modules

### ISSUE THREAD

**Issue by csvan** [TheBrainFamily/cypress-cucumber-preprocessor#450]
### Current behavior

When having `import` and `export` statements in step definitions, tests fail with:

`SyntaxError: 'import' and 'export' may appear only with 'sourceType: module'`

This is thrown for `.feature` files.

This has not happened in the past, but seems to have become an issue after upgrading to Cypress 5.

### Desired behavior

Tests with ES6 modules should pass

### Test code to reproduce

```
const cucumber = require('cypress-cucumber-preprocessor').default;

// / <reference types="cypress" />
// ***********************************************************
// This example plugins/index.js can be used to load plugins
//
// You can change the location of this file or turn off loading
// the plugins file with the 'pluginsFile' configuration option.
//
// You can read more here:
// https://on.cypress.io/plugins-guide
// ***********************************************************

// This function is called when a project is opened or re-opened (e.g. due to
// the project's config changing)

/**
 * @type {Cypress.PluginConfig}
 */
module.exports = (on) => {
  on('file:preprocessor', cucumber());
};
```

### Versions

* **Cypress version**:
5.2
* **Preprocessor version**:
2.5.5
* **Node version**:
14.11

**Comment by MattiaVio** [comment]
have the same issue, any hints ?

**Comment by csvan** [comment]
It seems like this was a temporary regression introduced in Cypress 5.2 specifically. Problem does not occur in Cypress 5.1 or the newly released 5.3. Will reopen if it reappears.
